src/render/font-metrics.ts

```diff
@@ -38,6 +38,7 @@ export class FontMetrics {
         img.style.margin = '0';
         img.style.padding = '0';
         img.style.verticalAlign = 'baseline';
+        img.style.display = 'inline';
```

Comment:
`initial` might be a better choice?

Reply:
@dohooo IE 11 doesn't have support for `initial` (https://caniuse.com/css-initial-value)

Reply:
wow! thanks for your reminding!